config/prod/s3-synapse-sync.yaml

```diff
@@ -21,3 +21,6 @@ parameters:
     }
   KmsDecryptPolicyArn: !stack_output_external "htan-synapse-sync-kms-key::KmsDecryptPolicyArn"
   BucketNamePrefix: "htan-dcc-*"
+  DockerImage: "clarisselau/minerva-batch"
+  Subnets: "subnet-1a2b3c4d,subnet-1a2b3c4d"
```

Comment:
You can use [sceptre stack_output_external reference](https://sceptre.cloudreach.com/2.2.1/docs/resolvers.html#stack-output-external) to access our VPC info, like so..

```
Subnets:
  - !stack_output_external computevpc::PrivateSubnet
  - !stack_output_external computevpc::PrivateSubnet1
  - !stack_output_external computevpc::PrivateSubnet2
```

which is NOT a string of subnets but rather a list of subnets (i.e. ["subnet-11111","subnet-22222","subnet-3333"])

Reply:
thanks! added the subnets